components/modal/Modal.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: Modal.js
+ * Component Name: Modal
+ * Project: design-system
+ * Date: Sun 10/12/2023 - 21:21
+ */
+
+import React from 'react';
+import Button from '../button/Button';
+import { IModalComponentProps } from './modal.interface';
+import { EIConTypes } from '../icon/icon.enum';
+import {
+  EButtonIconPosition,
+  EButtonSizes,
+  EButtonTypes,
+} from '../button/button.enum';
+import {
+  actions,
+  content,
+  header,
+  overlay,
+  title as titleClasses,
+  topContainer,
+} from './modal-css';
+
+const Modal = ({
+  children,
+  onSave,
+  onCancel,
+  active,
+  title,
+}: IModalComponentProps) => {
+  return (
+    <div className={`c-modal ${active ? overlay.join(' ') : 'hidden'}`}>
```

Comment:
What do you think of using the <dialog> html element for the modal? It would provide better accessibility, some built-in features to close / open the modal and some stylings: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

Reply:
yes, we could do it. never used before ;-)
Lets do it as an improvement after the presentation. you can open an issue. 
if you want to have it right away, then just let me know and I will update it later on.

Reply:
It is okay for me to do it after the presentation. I created a follow up issue: https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/43